Doberman/Longwatch.py

```diff
@@ -1,30 +1,37 @@
 #!/usr/bin/env python3
+"""
+What is this file? Should it be deleted?
```

Comment:
This was a simple way for users to monitor logging output without needing to consult the system logging utils